tests/unit/Core/Checkout/Cart/ProductCartProcessorTest.php

```diff
@@ -170,4 +171,42 @@ public function testPayloadIsTranslated(): void
         static::assertArrayHasKey('foo', $field);
         static::assertSame('baz', $field['foo']);
     }
+
+    public function testCoverIsRemovedIfProductMediaIsMissing(): void
+    {
+        $cart = new Cart('test');
+        $lineItem = new LineItem('A', 'product', 'A');
+        $lineItem->setCover(new MediaEntity());
+
+        $cart->setLineItems(new LineItemCollection([$lineItem]));
+
+        $product = (new SalesChannelProductEntity())->assign([
+            'id' => 'A',
+            'calculatedPrice' => new EmptyPrice(),
+            'calculatedPrices' => new PriceCollection(),
+            'calculatedMaxPurchase' => 1,
+            'productNumber' => 'A',
+            'stock' => 1,
+        ]);
+
+        $processor = new ProductCartProcessor(
+            $this->createMock(ProductGateway::class),
+            $this->createMock(QuantityPriceCalculator::class),
+            $this->createMock(ProductFeatureBuilder::class),
+            $this->createMock(ProductPriceCalculator::class),
+            $this->createMock(EntityCacheKeyGenerator::class),
+            $this->createMock(Connection::class)
+        );
+
+        $context = $this->createMock(SalesChannelContext::class);
+
+        $data = new CartDataCollection();
+        $data->set('product-A', $product);
+
+        static::assertInstanceOf(MediaEntity::class, $cart->getLineItems()->get('A')->getCover());
```

Comment:
```suggestion
        static::assertInstanceOf(MediaEntity::class, $cart->getLineItems()?->get('A')->getCover());
```

this should fix PHPStan

Reply:
Thank you :-)